server/controllers/LoanController.js

```diff
@@ -236,7 +236,24 @@ const loanController = {
     } else {
       statusMessageFunction(res, 400, `${loanStatus.badRequestMessage}` )
     }
-  }
-  
+  },
+  async allRepaymentLoan (req, res){
+    try{
+      const {rows} = await pool.query(queryTable.getAllRepayments)
+      if (rows.length === 0) {
```

Comment:
This can be improved to be `if(!rows.length)`

Reply:
admin should be able to view loan repayment and user should.